public/js/countyGraph.js

```diff
@@ -0,0 +1,190 @@
+const graph = document.getElementById('graph').getContext('2d');
+Chart.defaults.global.defaultFontSize = 10;
+
+const countyGraphChange = async (county) => {
+  const kenyaCounties = [
+    'Nairobi',
```

Comment:
Why are we hard-coding this list here and also fetch the same list via `await getCounties()` below?

Reply:
Fixed